entrypoint.py

```diff
@@ -454,61 +490,94 @@ def get_parents(path: Path, n: int) -> Path:
     subworkflows_path = Path(root_path, "subworkflows")
     workflows_path = Path(root_path, "workflows")
 
-    # Parse nf-test files for target test tags
+    logging.info(
+        f"Getting files that are different between {args.head_ref} and {args.base_ref}"
+    )
     changed_files = find_changed_files(
         root_path, args.head_ref, args.base_ref, args.ignored_files
     )
+    logging.debug(f"Found changed files:{changed_files}")
 
     # If an additional include YAML is added, we detect additional changed dirs to include
     if args.include:
+        logging.debug(f"Reading include file: {args.include}")
         include_files = read_yaml_inverted(args.include)
+        logging.info(f"Supplementing changed files with include files: {include_files}")
         changed_files = changed_files + detect_include_files(
             changed_files, include_files
         )
-    nf_test_files = detect_files(changed_files, "*.nf.test")
-    changed_component_names, changed_component_files = (
-        convert_nf_test_files_to_test_types(nf_test_files)
-    )
 
-    # Get only relevant results (specified by -t)
-    target_tests = get_target_tests(changed_component_names, args.types)
-    target_test_paths = get_target_tests(changed_component_files, args.types)
+    logging.info("Parsing nf-test files...")
+    nf_test_objects = [
+        NfTest(_nf_test_file, repo=root_path)
+        for _nf_test_file in detect_files([root_path], "*.nf.test")
+    ]
+    logging.debug(f"Found nf-test files: {[str(x.test_path) for x in nf_test_objects]}")
 
-    # Parse nf-test files to identify nf-tests containing "setup" with changed module/subworkflow/workflow
-    nf_test_changed_setup = find_nf_tests_with_changed_dependencies(
-        [modules_path, subworkflows_path, workflows_path],
-        target_tests,
+    logging.info(
+        "Getting intersect of changed files and Nextflow components with nf-test files..."
+    )
+    # Get intersect of changed files and Nextflow components with nf-test files
+    logging.info("Finding Nextflow components which have been modified...")
+    directly_modified_nf_tests = [
+        nf_test_object
+        for changed_file in changed_files
+        for nf_test_object in nf_test_objects
+        if nf_test_object.detect_if_path_is_in_test(changed_file)
+    ]
+    logging.debug(
+        f"Directly modified nf-tests: {[str(x.test_path) for x in directly_modified_nf_tests]}"
```

Comment:
```suggestion
        f"nf-tests with directly modified nf-test scripts, nextflow scripts, or config files: {[str(x.test_path) for x in directly_modified_nf_tests]}"
```
Maybe just to be more explicit about what's included?